test/test_plugin_helper.rb

```diff
@@ -4,6 +4,9 @@
 require 'webmock/minitest'
 require 'webmock'
 
+FactoryBot.definition_file_paths << File.join(File.dirname(__FILE__), 'factories')
+FactoryBot.reload
+
```

Comment:
Ditto. You can see how VMWare is implemented. There is no difference except VMWare is part of core and Proxmox is part of plugin. If plugin is not found, tests should be skipped.

Reply:
I think we are good to go. 
@timogoebel / @lzap / @ekohl ?

Reply:
If you are able to reuse factories from the plugin that would be better.